### PR TITLE
Refine some details and coding style for Rust codes

### PR DESCRIPTION
**Pull request by sjinzh**
If this PR is related to coding or code translation, please read the [contribution guideline](https://github.com/krahets/hello-algo/issues/15#issue-1464069591), fill out the checklist, and paste the console outputs to the PR.

- [x] I've tested the code and ensured the outputs are the same as the outputs of reference codes.
- [x] I've checked the codes (formatting, comments, indentation, file header, etc) carefully.
- [x] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

**Review by krahets — COMMENTED** [review]
Thank @sjinzh for the great improvement!

I recommend keeping the comment format of `/* ... */` for the class and function headers.

This book follow this rule of comment formatting.

```
/* 标题注释，用于标注函数、类、测试样例等 */

// 内容注释，用于详解代码

/**
 * 多行
 * 注释
 */
```

More details at [关于本书](https://www.hello-algo.com/chapter_preface/about_the_book/#014)

File: codes/zig/chapter_stack_and_queue/array_stack.zig [hunk 1]
```diff
@@ -90,7 +90,7 @@ pub fn main() !void {
     std.debug.print("\n栈的长度 size = {}", .{size});
 
     // 判断栈是否为空
-    var is_empty = stack.empty();
+    var is_empty = stack.is_empty();
```
Comment (krahets) [inline codes/zig/chapter_stack_and_queue/array_stack.zig:93]: isEmpty()

Comment (krahets) [reply codes/zig/chapter_stack_and_queue/array_stack.zig:93]: I found `is_empty()` in array_stack.zig but `empty()` in linkedlist_stack.zig

Please unify the function naming.

Comment (krahets) [reply codes/zig/chapter_stack_and_queue/array_stack.zig:93]: I found is_empty() in array_stack.zig but empty() in linkedlist_stack.zig

Please unify the function naming.

**Review by krahets — COMMENTED** [review]
Two more comments, please.

**Comment by sjinzh** [comment]
> Two more comments, please.

The below comment style in the rust file header would result in rustdoc warning with the following include!() macro.
> ```
> /*
>  * 多行
>  * 注释
>  */
> ```

File: codes/zig/chapter_stack_and_queue/array_stack.zig (hunk 1 above)
Comment (sjinzh) [reply codes/zig/chapter_stack_and_queue/array_stack.zig]: done.

**Review by krahets — APPROVED** [review]
Thanks @sjinzh ! Now merge it.

**Pull request merged**